challenges/objects-arrays.js

```diff
@@ -110,12 +175,15 @@ The zoos need to know their total animal population across the United States.  F
 
 */
 const populationTotal;
+
+const populationTotal = zooAnimals.reduce((a, b) => a + b.population, 0);
```

Comment:
You're declaring the constant `populationTotal` twice, which has two problems: 1. Declaring a variable twice and 2. Trying to redefine a constant.